problems/my_select.rb

```diff
@@ -1,5 +1,8 @@
 class Array
-  def my_select(&prc)
+  # Define a method `my_select(&prc)` that correctly returns
+  # an array of selected elements according to the block.
+  # You CANNOT use Ruby's `Array#select` or `Array#select!`.
```

Comment:
Let's also disallow `Array#reject` and `Array#reject!` - update in specs/solutions as well